samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -63,6 +72,14 @@ enum mgmt_cb_return CommandHandler(uint32_t event,
 	return MGMT_CB_OK;
 }
 
+enum mgmt_cb_return DfuStoppedHandler(uint32_t event, enum mgmt_cb_return prev_status, int32_t *rc, uint16_t *group,
```

Comment:
Although our compiler does not complain about it, maybe it's still better to tag all arguments are unused or skip their names.

Reply:
Ok, I remove the unused argument names, also for the already existing callbacks.